frontend/components/Slider/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from '@shopgate/pwa-common/components/Link';
+import Slider from '@shopgate/pwa-common/components/Slider';
+import styles from './style';
+
+/**
+ * Renders the component.
+ * @param {Object} props The component props.
+ * @returns {JSX}
+ */
+const SliderWidget = (props) => {
+  /**
+  * Creates an Item for a Slide.
+  * @param {Object} slideItem The Slide Item data.
+  * @return {JSX} The Slide Item.
+  */
+  const createSliderItem = (slideItem) => {
+    if (slideItem.img === '' || slideItem.img === undefined) {
```

Comment:
check for typeof object or string here and add comments to the different ifs

Reply:
only check for string needed. done.